packages/realm/src/Logger.ts

```diff
@@ -161,27 +161,29 @@ export type Logger = (level: NumericLogLevel, message: string) => void;
 
 /**
  * A callback passed to `Realm.setLogger`.
- *
  * @param level   - The level of the log entry.
  * @param message - The message of the log entry.
  * @since 12.0.0
  * @deprecated Will be removed in v13.0.0
  */
 export type LoggerCallback1 = (level: LogLevel, message: string) => void;
-export type LoggerCallbackArgs = {
+
+/**
+ * Represents an entry in the log.
+ */
+export type LogEntry = {
```

Comment:
I believe "entry" is semantically more precise.

Reply:
Yes, it reads better